src/DXRModules/DeusEx/Classes/DXRBacktracking.uc

```diff
@@ -471,12 +473,16 @@ function VandCmdAnyEntry()
     ConversationFrobOnly(GetConversation('M12JockFinal2'));
 
     flags = dxr.flagbase;
-    flags.SetBool('TongTrigger_Played', false,, 1);
+    flags.SetBool('TongTrigger_Played', false,, 15);
```

Comment:
really this makes no different, it's setting to false, this could be swapped to a DeleteFlag